### DISCUSSION TITLE
Startup Failure (Windows 11 Location Permission)

### DISCUSSION THREAD

**Discussion by Kiyo004** [SafeExamBrowser/seb-win-refactoring#928] [Q&A] [answered]
[2024-06-25_12h58m53s_Client.log](https://github.com/user-attachments/files/15965579/2024-06-25_12h58m53s_Client.log)
[2024-06-25_12h58m53s_Runtime.log](https://github.com/user-attachments/files/15965581/2024-06-25_12h58m53s_Runtime.log)

I did everything, reinstalling windows, close and reopen the services.  checked for any strange apps, all clean.
My university IT is also confused of what's wrong with my system or the software itself. 

Please help me urgently, i have an important exam to take in 2 days.

**Comment by dbuechel** [comment, marked as answer]
There appears to be an issue with the initialization of the (wireless) network adapter:

```
   Exception Message: Access is denied
   Exception Type: System.ComponentModel.Win32Exception

   at SimpleWifi.Win32.WlanClient..ctor()
   at SimpleWifi.Wifi..ctor()
   at SafeExamBrowser.SystemComponents.Network.NetworkAdapter.Initialize() in C:\Users\appveyor\projects\seb-win-refactoring-ist041kp6jai2xxx\SafeExamBrowser.SystemComponents\Network\NetworkAdapter.cs:line 92
   at SafeExamBrowser.Client.Operations.ShellOperation.InitializeSystemComponents() in C:\Users\appveyor\projects\seb-win-refactoring-ist041kp6jai2xxx\SafeExamBrowser.Client\Operations\ShellOperation.cs:line 214
   at SafeExamBrowser.Client.Operations.ShellOperation.Perform() in C:\Users\appveyor\projects\seb-win-refactoring-ist041kp6jai2xxx\SafeExamBrowser.Client\Operations\ShellOperation.cs:line 81
   at SafeExamBrowser.Core.OperationModel.OperationSequence.Perform() in C:\Users\appveyor\projects\seb-win-refactoring-ist041kp6jai2xxx\SafeExamBrowser.Core\OperationModel\OperationSequence.cs:line 108
```

First, please try to update to the latest available release version (currently 3.7.1). Then retry, and should it still fail with the same error message, ensure that you have given SEB permission to access the location services (see Settings > Privacy & security > Location in the system control panel of Windows).

**Comment by Kiyo004** [comment]
Thank you so much, turning on Location works.

**Reply by dbuechel** [reply]
Okay, thanks for the update. That is a bit strange, as supposedly the new access restriction should not yet be rolled out, but then again, it's Microsoft...

**Reply by dbuechel** [reply]
> You'll then have until the following Windows feature release in fall of 2024 to test your app's scenarios, and to make any necessary changes. After that date, Windows devices will begin getting the update that contains the behavior change described above.
>
> https://learn.microsoft.com/en-us/windows/win32/nativewifi/wi-fi-access-location-changes

**Reply by Kiyo004** [reply]
> Okay, thanks for the update. That is a bit strange, as supposedly the new access restriction should not yet be rolled out, but then again, it's Microsoft...

we should inform this to the dev team, i'm currently on 24h2 of windows 11

**Reply by dbuechel** [reply]
Aha, okay, but then you're using an insider preview build?

**Comment by dbuechel** [comment]
I'll leave this open for the time being, most likely other users will encounter the issue as well.

**Comment by R2D2byBest** [comment]
Hi,

I have 2 users with the same error

`2024-10-29 11:06:16.940 [01] - ERROR: Caught unexpected exception while performing operation 'LazyInitializationOperation'!

   Exception Message: Accès refusé
   Exception Type: System.ComponentModel.Win32Exception

   à SimpleWifi.Win32.WlanClient..ctor()
   à SimpleWifi.Wifi..ctor()
   à SafeExamBrowser.SystemComponents.Network.NetworkAdapter.Initialize() dans C:\Users\appveyor\projects\seb-win-refactoring-mltg1wfbmuoivnbc\SafeExamBrowser.SystemComponents\Network\NetworkAdapter.cs:ligne 92
   à SafeExamBrowser.Client.Operations.ShellOperation.InitializeSystemComponents() dans C:\Users\appveyor\projects\seb-win-refactoring-mltg1wfbmuoivnbc\SafeExamBrowser.Client\Operations\ShellOperation.cs:ligne 230
   à SafeExamBrowser.Client.Operations.ShellOperation.Perform() dans C:\Users\appveyor\projects\seb-win-refactoring-mltg1wfbmuoivnbc\SafeExamBrowser.Client\Operations\ShellOperation.cs:ligne 85
   à SafeExamBrowser.Core.OperationModel.OperationSequence.Perform() dans C:\Users\appveyor\projects\seb-win-refactoring-mltg1wfbmuoivnbc\SafeExamBrowser.Core\OperationModel\OperationSequence.cs:ligne 108`

Regards

**Reply by dbuechel** [reply]
As explained above:

> [E]nsure that you have given SEB permission to access the location services (see Settings > Privacy & security > Location in the system control panel of Windows).

**Comment by Aditya-855** [comment]
Hey i m facing error on the Demo Test of the SEB Exam Browser to check My System Compatibility (-105 , main request)
![SEB Error](https://github.com/user-attachments/assets/2be8c94a-dc4a-49d5-a8d0-975db07b3cdd)
